src/pages/Home.tsx

```diff
@@ -77,45 +79,51 @@ export default function Home() {
         </div>
       </section>
 
-      {/* 채널별 섹션 */}
-      {Object.entries(channelPosts).map(([channelId, { posts, name }]: any) => (
-        <section key={channelId}>
-          <div className="mb-3">
-            <span className="text-2xl font-semibold">최근 </span>
-            <span className="text-[#fcc404] text-2xl font-semibold">
-              {name}
-            </span>
-            <span className="text-2xl font-semibold"> 포스트</span>
-          </div>
-          <div className="flex flex-wrap gap-10">
-            {posts.length > 0 ? (
-              posts
-                .filter((post: any) => post.title.includes("{")) // 중괄호 포함된 title만 필터링
-                .map((post: any) => {
-                  const parsedTitle = JSON.parse(post.title);
-                  return (
-                    <RecentPosts
-                      key={post._id}
-                      title={parsedTitle.title}
-                      youtubeTitle={parsedTitle.contents}
-                      youtubeThumbnail={parsedTitle.image || "/logo.png"}
-                      avatarImg={post.author?.image || "/logo.png"}
-                      channelName={name}
-                      postId={post._id}
-                      userId={post.author?._id}
-                      userName={post.author?.fullName}
-                    />
-                  );
-                })
-            ) : (
-              <p>등록 되어있는 포스트가 없습니다...🫠</p>
-            )}
-          </div>
-        </section>
-      ))}
+      {/* 채널별 섹션 및 광고 */}
+      {Object.entries(channelPosts).map(
+        ([channelId, { posts, name }]: [string, ChannelPosts], index) => (
+          <div key={channelId}>
```

Comment:
사이드바에서 이미 같은 key값을 사용하고 있어서, 다음과 같이 key 값을 조금 다르게 주면 더 좋을 것 같아요!
```suggestion
          <div key={`recent-${channelId}`}>
```